opendevin/sandbox/docker/ssh_box.py

```diff
@@ -176,7 +176,9 @@ def start_ssh_session(self):
         else:
             username = 'root'
         logger.info(
-            f"Connecting to {username}@{hostname} via ssh. If you encounter any issues, you can try `ssh -v -p {self._ssh_port} {username}@{hostname}` with the password '{self._ssh_password}' and report the issue on GitHub."
+            f"Connecting to {username}@{hostname} via ssh. "
```

Comment:
@Borda This shouldn't pass CI linter test as it's using double quotes instead of single quotes... but it does pass CI.

I verified locally that a revert of your e32d95cb would report this as lint error. Could you please check this? Otherwise we may have to revert that PR.

Reply:
Sure, let me check it on Monday

Reply:
well, found that format was not ever called correctly; see #1457
but still not resolved fully the single-quote issues yet

Reply:
The last resort would be to add `flake8` back, maybe after `ruff`, since `flake8` only detects but doesn't do the fix.

Reply:
reported in https://github.com/astral-sh/ruff/issues/11209

Reply:
Apparently they intentionally don't want to enforce quote behavior here. I think that's fine. We already disabled black's quoting changes in `pyproject.toml`, so we won't see people changing this line from single to double and from double to single endlessly (unless they manually do so).